Formula/simple-obfs.rb

```diff
@@ -13,12 +13,23 @@ class SimpleObfs < Formula
 
   depends_on "asciidoc" => :build
   depends_on "xmlto" => :build
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+  depends_on "udns"
+  depends_on "libev"
+  depends_on "libsodium"
   depends_on "pcre"
   depends_on "openssl"
 
   def install
     ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
 
+    system "git", "clone", "https://github.com/shadowsocks/libcork.git", "libcork/"
+    Dir.chdir("libcork")
+    system "git", "checkout", "3bcb8324431d3bd4be5e4ff2a4323b455c8d5409"
```

Comment:
We won't be able to accept that… dependencies and resources should only be downloaded through homebrew's mechanism, and not through manual use of `git`.